Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1384,7 +1384,7 @@ extension BrowserViewController: WKNavigationDelegate {
 
         tab.expireSnackbars()
 
-        if let url = webView.URL where !ErrorPageHelper.isErrorPageURL(url) {
+        if let url = webView.URL where !ErrorPageHelper.isErrorPageURL(url) && url.scheme != "about" {
```

Comment:
Yeah I don't see any references to `about:` in the current code anymore. For example `about:home` has now become `http://localhost:1234/about/home/#panel=0` instead.